process/factory/interceptorscontainer/metaInterceptorsContainerFactory_test.go

```diff
@@ -420,7 +420,7 @@ func TestMetaInterceptorsContainerFactory_With4ShardsShouldWork(t *testing.T) {
 	numInterceptorsMetablock := 1
 	numInterceptorsShardHeadersForMetachain := noOfShards
 	numInterceptorsTransactionsForMetachain := noOfShards + 1
-	numInterceptorsMiniBlocksForMetachain := noOfShards + 1
+	numInterceptorsMiniBlocksForMetachain := noOfShards + 1 + 1
```

Comment:
+2 (as that is how was used in the other tests)

Reply:
fixed